zcash_client_sqlite/src/lib.rs

```diff
@@ -220,14 +234,25 @@ impl<C: Borrow<rusqlite::Connection>, P: consensus::Parameters> InputSource for
         anchor_height: BlockHeight,
         exclude: &[Self::NoteRef],
     ) -> Result<Vec<ReceivedNote<Self::NoteRef, Note>>, Self::Error> {
-        wallet::sapling::select_spendable_sapling_notes(
+        let received_iter = std::iter::empty();
+        let received_iter = received_iter.chain(wallet::sapling::select_spendable_sapling_notes(
             self.conn.borrow(),
             &self.params,
             account,
             target_value,
             anchor_height,
             exclude,
-        )
+        )?);
+        #[cfg(feature = "orchard")]
+        let received_iter = received_iter.chain(wallet::orchard::select_spendable_orchard_notes(
```

Comment:
This should be fixed by the third commit of https://github.com/zcash/librustzcash/pull/1244 (which I'm in the process of extracting out on its own.)

Reply:
Whether or not we include the third commit of #1244 in Zashi 1.0 (and we might not), we should fix this because it's a bug in the current implementation.

Reply:
Opened #1260.